sklearn/feature_selection/from_model.py

```diff
@@ -183,10 +187,28 @@ class SelectFromModel(BaseEstimator, SelectorMixin):
     `threshold_`: float
         The threshold value used for feature selection.
     """
-    def __init__(self, estimator, threshold=None, prefit=False):
+    def __init__(self, estimator, threshold=None, prefit=False,
+                 max_n_features=None):
```

Comment:
Regardless of supporting float, I think you're right that this should be renamed to `max_features`. That's consistent with numerous `max_features` parameters in the scikit-learn codebase.